iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractBranchAdder.java

```diff
@@ -110,4 +121,24 @@ protected VoltageLevelExt checkAndGetVoltageLevel2() {
         return voltageLevel2;
     }
 
+    private String checkAndGetDefaultVoltageLevelId(String connectableBus) {
+        if (connectableBus != null) {
+            VoltageLevelExt defaultVoltageLevel = (VoltageLevelExt) getNetwork().getBusBreakerView()
```

Comment:
In case of a string corresponding to connectableBus that is not an id of a bus, what is happening ? In open-loadflow, we always check that the bus is not null. Why we don't need it here?

Reply:
Regarding the first question : if no voltage level is given and connectableBus is not a bus ID, the creation fails with a NullPointerException (when trying to getVoltageLevel() on the not found bus). Should I change this to make the origin of the error clearer ?

Regarding the second point : if I understood right, the bus is checked after when building the Terminal and is primarily used to say if the element is really connected to the connectableBus (if the bus is null, the element is not connected).

Reply:
Regarding the first question, I would prefer a clearer error than a NPE indeed. I put in my review that voltageLevel should return null in this case but we can go even deeper and state that the bus associated to the id does not exist.